runtime/gc_glue_java/MarkingDelegate.cpp

```diff
@@ -556,7 +569,7 @@ MM_MarkingDelegate::getReferenceStatus(MM_EnvironmentBase *env, omrobjectptr_t o
 		referentMustBeCleared = (0 != (referenceObjectOptions & MM_CycleState::references_clear_soft));
 		*referentMustBeMarked = *referentMustBeMarked || (
 			((0 == (referenceObjectOptions & MM_CycleState::references_soft_as_weak))
-			&& ((UDATA)J9GC_J9VMJAVALANGSOFTREFERENCE_AGE(env, objectPtr) < _extensions->getDynamicMaxSoftReferenceAge())));
+			&& ((uintptr_t)J9GC_J9VMJAVALANGSOFTREFERENCE_AGE(env, objectPtr) < _extensions->getDynamicMaxSoftReferenceAge())));
```

Comment:
Existing problem, but seems to me that dynamic/max age we have in Extensions should be u32, not udata.
Leave it as-is for now, but try to remember to fix this (there is probably a dozen to files to update) at some point.